docs/application/tizen-studio-0/common-tools/emulator-manager.md

```diff
@@ -56,7 +56,7 @@ To create a new emulator instance:
 
    > **Note**
    >
-   > To run the application faster, switch on CPU VT and GPU. If CPU VT is disabled, check [Increasing the Application Execution Speed](emulator.md#speed) for more information. If GPU is disabled, [install the latest vendor-provided graphic driver](../setup/prerequisites.md#emulator).
+   > To run the application faster, switch on CPU VT and GPU. If CPU VT is disabled, check [Increasing the Application Execution Speed](emulator.md#speed) for more information. If GPU is disabled, [install the latest vendor-provided graphic driver](../setup/additional-requirements.md#tizen-studio-emulator-requirements).
```

Comment:
There is no hash tag for tizen-studio-emulator-requirements.

`#tizen-studio-emulator` is correct.

Reply:
Done